### ISSUE TITLE
Website issues

### ISSUE THREAD

**Issue by FrancescaPanini** [SChAth/ScAthiop#173]
Hello,
I am experiencing the following problems when I look for some records I created in order to check that they look in order:
1) In the record for Gondar there are missing spaces between roleName and persName in the preview but I checked the xml record and the spaces should be there...
2) I created a new record for Gondarine Kingdom but, when I search in the website, it does not come up. If I try to follow the linked placeName inside the record for Gondar, the website redirects me to a different record instead of the Gondarine kingdom record
3) Yesterday I completed the record for LOC2248DabraB but again, when I look for it in the website it does not come up. 

Thanks

**Comment by PietroLiuzzo** [comment]
1 should be now fixed
2 I cannot reproduce this issue. I am correctly directed to an empty record
3 I cannot reproduce this issue. I see the latest changes recorded in GIT also in the website

Please wait few seconds before checking, it might simply be that the connection's upload speed (?) does not let you see immediately the changes. Also, refresh pages from the server holding shift while hitting the refresh button in your browser or F5 if you use that.

**Comment by FrancescaPanini** [comment]
OK, 
1) I see it is now fixed, thanks
2) I still cannot find the record if I search and if i follow the link inside Gondar's record I am redirected to an empty record regarding a stub rather than to the completed Gondarine kingdom record (LOC6554GondarineK)
3) I can now find it when i search but the title show also the English translation as this place name has 2 @corresp. I noticed that this is an issue with all the place names which have 2 @correps (usually one is the normalised name and the other is the English translation, see for instance Gannata ʾIyasus).

Hope it makes sense.

Thanks
